src/main/java/com/conveyal/datatools/manager/models/Project.java

```diff
@@ -129,4 +130,17 @@ public void delete() {
         @JsonProperty("otpServers")
         public abstract Collection<OtpServer> availableOtpServers ();
     }
+
+    /**
+     * A MixIn to be applied to this project, for returning a single project, so that the list of otpServers is
+     * included in the JSON response.
+     *
+     * Usually a mixin would be used on an external class, but since we are changing one thing about a single class, it
+     * seemed unnecessary to define a new view.
+     */
+    public abstract static class ProjectWithoutOtpServers {
```

Comment:
I don't understand why this is needed. Isn't `otpServers` omitted by default? The Mixin above is only needed to add the `otpServers` property.

Reply:
Ah, you're right. Not sure why I added it.